lib/src/translations/tolgee_remote_translations.dart

```diff
@@ -11,21 +11,18 @@ import '../api/tolgee_project_language.dart';
 
 String normalizeLanguageCode(String languageCode) {
   // Split the language code by underscore
-  List<String> parts = languageCode.split('_');
+  List<String> parts = languageCode.split('-');
```

Comment:
Are you sure that you don't need to check for the "_" as well?

Reply:
I`m sure that we don`t need to check for the"_". Also, Vitalii checked it on the iPhone and everything works fine

Reply:
On my testing android phone, I received it with "_", so please check it for both, just to be sure.